ols/src/tools/oc_cli.py

```diff
@@ -38,6 +42,15 @@ def sanitize_oc_args(args: list[str]) -> list[str]:
         if safe_part:  # only add non-empty results
             sanitized_args.append("".join(safe_part).strip())
 
+    # Temporary workaround for granite.
+    # Sometimes within the list we may get two args combined; ex: [top pod]
+    sanitized_args = " ".join(sanitized_args).split(" ")
```

Comment:
How "temporary" is this? :)
Would it be improved with more instructions? Newer version?